plugins/modules/keycloak_clientscope.py

```diff
@@ -319,6 +335,9 @@ def normalise_cr(clientscoperep, remove_ids=False):
             if remove_ids:
                 mapper.pop("id", None)
 
+            for key, value in mapper.items():
+                mapper[key] = normalise_boolean(value)
```

Comment:
If a string entry for some reason equals "true" or similar, it will now be converted to a boolean, which isn't right either. I think you need to limit this to a specific set of keys that usually have boolean values.